skins/wmde19/src/pages/error.ts

```diff
@@ -1,9 +1,12 @@
 import Vue from 'vue';
 import VueI18n from 'vue-i18n';
-import Error from '@/components/pages/Error.vue';
-import App from '@/components/App.vue';
 import PageDataInitializer from '@/page_data_initializer';
 import { DEFAULT_LOCALE } from '@/locales';
+import App from '@/components/App.vue';
+
+import Component from '@/components/pages/Error.vue';
```

Comment:
Why `Component` and not `Error` ?

Reply:
The idea was to keep the file as "general" as possible because this file will need to be copy-pasted for each entry point. That way, all you would need to change in between files is the "Error.vue" file name while the rest of the file can stay the same. It's a bit weird and I really don't like the redundancy, but I cannot think of a good way to alleviate this right now.